pkg/util/filter/filter_function.go

```diff
@@ -1,3 +1,5 @@
 package filter
 
+// Func is a function type that is the signature of a filter function used in log deletion.
+// This is used as a parameter in the Rebound function.
 type Func func(string) bool
```

Comment:
WDYT of naming it `Predicate`? I think it is how Apache Spark and Java name a similar thing.